lessons/Algebra2/logarithmic2-building-models/langs/en-us/pages/from-expt-to-log.adoc

```diff
@@ -0,0 +1,98 @@
+= From Exponents to Logs
+
+++++
+<style>
+/* Make autonums bold for matching pages */
+td .autonum:after { content: ')' !important; }
+td:first-child { padding-left: 0; }
+
+/* Add a top-margin to tables */
+table { margin-top: 1ex; }
+
+table:last-child img { height: 1.8in; }
+</style>
+++++
+
+== Connecting to Prior Knowledge
+
+- The first scatter plot below shows an *exponential relationship*: the number of Covid Infections as a function of Days
+- The second scatter plot is the same data with the x- and y-axes _swapped_: Days as a function of Infections
```

Comment:
Suggested rewrite of line 19:
The second scatter plot shows a *logarithmic relationship* - the same data with the x- and y-axes _swapped_: Days as a function of Infections